src/System.Memory/tests/ReadOnlySpan/IndexOfAny.byte.cs

```diff
@@ -32,11 +32,11 @@ public static partial class ReadOnlySpanTests
         public static void IndexOfAnyStrings_Byte(string raw, string search, char expectResult, int expectIndex)
         {
             byte[] buffers = Encoding.UTF8.GetBytes(raw);
-            var span = new ReadOnlySpan<byte>(buffers);
+            ReadOnlySpan<byte> span = new ReadOnlySpan<byte>(buffers);
```

Comment:
FWIW, this was fine as `var`.  Our guidelines just outlaw it when the type isn't named on the RHS, which generally means something other than a new or a cast.  But naming it in such situations is fine, too.